models/Translation/Dao/DaoInterface.php

```diff
@@ -0,0 +1,23 @@
+<?php
+/**
+ * Pimcore
+ *
+ * This source file is available under two different licenses:
+ * - GNU General Public License version 3 (GPLv3)
+ * - Pimcore Enterprise License (PEL)
+ * Full copyright and license information is available in
+ * LICENSE.md which is distributed with this source code.
+ *
+ * @category   Pimcore
+ * @package    Translation
+ *
+ * @copyright  Copyright (c) Pimcore GmbH (http://www.pimcore.org)
+ * @license    http://www.pimcore.org/license     GPLv3 and PEL
+ */
+
+namespace Pimcore\Model\Translation\Dao;
+
+interface DaoInterface
```

Comment:
I think there's no reason for keeping this interface, as `getTableName()` is now anyway generic and based on the domain.

Reply:
Kicked it.